app/templates/application.plush.html

```diff
@@ -6,6 +6,8 @@
   <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0">
   <meta name="csrf-param" content="authenticity_token" />
   <meta name="csrf-token" content="<%= authenticity_token %>" />
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```

Comment:
Let's remove these two. All JS should be contained in the assets folder. jQuery needs a little fix there to work.